README.md

```diff
@@ -47,6 +47,20 @@ website, backend, and associated dependent services.
 docker compose up --build
 ```
 
+<details>
+  <summary>If you are getting `unknown flag: --build` error. Upgrade docker compose using this script.</summary>
+   DOCKER_CONFIG=${DOCKER_CONFIG:/usr/local/lib/docker}
```

Comment:
Does this differ form the standard-way of installing the compose plugin? If not I would suggst that we link to https://docs.docker.com/compose/install/linux/ .. e.g. there are different ways to install docker and some people might prefer to use the apt..

(BTW the "rich diff" preview of github shows all the script-lines in one line, maybe because they are not within a code-block? Nevertheless I would prefer that we don't include install details for docker but rather link to the official docs if possible).

Reply:
> Does this differ form the standard-way of installing the compose plugin? If not I would suggest that we link to https://docs.docker.com/compose/install/linux/ .. e.g. there are different ways to install docker and some people might prefer to use the apt..


I was having this issue myself and I was not able to solve this by following the official documents. That's why i did it manually. In-fact I have installed and upgraded to very latest version of docker compose when I got this error in the first place.

------------------------------------



> (BTW the "rich diff" preview of github shows all the script-lines in one line, maybe because they are not within a code-block? Nevertheless I would prefer that we don't include install details for docker but rather link to the official docs if possible).

You are allowed to edit the PR code if you want. you can edit and put the code into code-block if you want. I am totally fine with it.

Reply:
I have added code-block as you told me.